lib/cmd.js

```diff
@@ -181,11 +181,7 @@ class Cmd {
       .option('--loglevel [loglevel]', 'level of logging to display ["error", "warn", "info", "debug", "trace"]', /^(error|warn|info|debug|trace)$/i, 'debug')
       .description('Upload your dapp to a decentralized storage (e.g embark upload ipfs).')
       .action(function (platform, env, _options) {
-        let environment = env || 'development';
-        embark.initConfig(environment, {
-          embarkConfig: 'embark.json', interceptLogs: false
-        });
-        _options.env = environment;
+        _options.env = env || 'development';
```

Comment:
why remove the initConfig? in fact I don't quite understand how ```this.config.storageConfig``` would work later without this